ext/natives/codegen.lua

```diff
@@ -281,67 +285,70 @@ function rpcEnvironment.context_rpc(nativeName)
 	rn.ctx = ctx
 	rn.args = args
 
-	removeServerNative(nativeName)
-	codeEnvironment.native(nativeName)
-		codeEnvironment.arguments(n.arguments)
-		codeEnvironment.apiset('server')
-		codeEnvironment.returns('void')
-		curNative.rpc = true
-		curNative.ogHash = n.hash
+	removeServerNative(n.name)
+	codeEnvironment.native(n.name)
+	codeEnvironment.arguments(n.arguments)
+	codeEnvironment.apiset('server')
+	codeEnvironment.returns('void')
+	for _, aliasName in ipairs(n.aliases) do
+		codeEnvironment.alias(aliasName)
```

Comment:
Including aliases will also require `removeServerNative(aliasName)`, same applies to the other cases `codeEnvironment` below. Otherwise the server files will contain duplicate definitions and MonoV2 won't compile.

Reply:
Can you double check after clearing all intermediate files, e.g., `rm -rf ext/natives/inp ext/natives/out`? MonoV2's compilation will give me, for example,

```cs
public static void SetPedHairColor(int ped, int colorID, int highlightColorID)
{
    NativesImpl.void_0xbb43f090(N64.Val(ped), N64.Val(colorID), N64.Val(highlightColorID));
}

// ...

public static void SetPedHairColor(int ped, int colorID, int highlightColorID) => SetPedHairTint(ped, colorID, highlightColorID);
```

Reply:
Ah. Apparently `native-decls/native_md` not being cleaned properly - leaving old definitions around (on both computers I tested).